app/controllers/super_admin/instance_statuses_controller.rb

```diff
@@ -5,6 +5,17 @@ def show
     sha
     postgres_status
     redis_metrics
+    chatwoot_edition
+  end
+
+  def chatwoot_edition
+    is_enterprise = if ChatwootApp.enterprise?
+                      'Enterprise'
+                    else
+                      'Community'
+                    end
+    is_custom = ChatwootApp.custom? ? 'Custom' : ''
+    @metrics['Chatwoot edition'] = "#{is_enterprise} #{is_custom}"
   end
 
```

Comment:
```suggestion
  def chatwoot_edition
    @metrics['Chatwoot edition'] = case
    when ChatwootApp.enterprise?
      'Enterprise'
    when ChatwootApp.custom?
      'Custom'
    else
      'Community'
    end
  end

```

Reply:
Make sense, Enterprise custom and Community custom are both custom. It's more simple, thanks! suggestion applied, PTAL.